SwanProjects/bin/swan_bash

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -i
+#Author Omar.Zapata@cern.ch 2021
+"""
+This script allows to start a bash interpreter inside a project environment.
+This is called from the extension SwanTerminal and it requires like a parameter the project name to
+build the path to the project with $HOME/SWAN_projects/$PROJECT to load the project environment.
+When the bash session is started we modify the prompt indicating in which project the terminal is located.
+Aditionally bash run completed isolated from other possible environments then we pass like parameters to the isolated environment some
+required basic environment variables such as:
+
+* HOME with path to the user home
+* PATH with default paths for the system
+* OAUTH2_TOKEN and OAUTH_INSPECTION_ENDPOINT required by EOS storage
+
+swan_env will load the other variables from the stack, inside the isolated enviroment.
+"""
+
+clear
+if ! [ -x "$(command -v jq)" ]; then
+  echo 'Error: jq is not installed.' >&2
+  sleep 60
+  exit 1
+fi
+if [[ $# -gt 0 ]] ; then
+    PROJECT=$1
+    PROJECT_PATH="$HOME/SWAN_projects/$PROJECT"
```

Comment:
now, this script required the project path, instead of only the project name,
thinking a little about the future implementation for projects anywhere.